### ISSUE TITLE
Documentation in English or Spanish

### ISSUE THREAD

**Issue by alesanfe** [alibaba/fastjson2#209]
It would be nice to have more documentation in English, or even in Spanish as it was write on the ISSUE Documentation in English or Spanish #4056 in the first version, since they are the two most spoken languajes.
Here is a basic traduction to the documentation in Spanish:

![image](https://user-images.githubusercontent.com/72869496/167433555-75146c79-0c80-4f72-a30d-88ba747c6224.png)

**Comment by wenshao** [comment]
Yes, there should be English documentation. This project has just started, and we are still focusing on the code itself, and the Chinese documentation is not perfect. You are welcome to help this project with documentation, including English and Spanish.

**Comment by alesanfe** [comment]
Okey, I recommend use the next translator:

https://www.deepl.com/es/translator-mobile/
